flict/flictlib/format/markdown_format.py

```diff
@@ -10,71 +10,183 @@
+            #            identified_license_aliased = package.get('identified_license_aliased', identified_license)
+            identified_license = package.get('outbound_license', None)
+            identified_license_aliased = package.get('outbound_license_aliased', None)
+
+            identified_licenses = package.get('outbound_licenses', None)
+            #identified_licenses_aliased = package.get('outbound_licenses_aliased', identified_licenses)
+
+            output.append(self.package_name_header(package['name']))
+            description = package.get('description', "")
+            if not description:
+                output.append(f"*Description*: {description}\n")
+
+            if identified_licenses == []:
+                output.append("Main license: Could not be identified, see problems section below.\n")
+                output.append(f"Declared license: {package['license']}\n")
+            else:
+                # Out of the outbound licenses, present the first and suggest the others
+                #identified_license = identified_licenses[0]
+                #identified_license_aliased = identified_licenses_aliased[0]
+                output.append(f"Main license: {identified_license}\n")
+
+                if identified_license != identified_license_aliased:
+                    output.append(f"Main license alias: {identified_license_aliased}\n")
+                if identified_licenses and len(identified_licenses) > 1:
+                    output.append(f"*Other possible main licenses*: {identified_license}\n".format(identified=", ".join(identified_licenses[1:])))
+
+            output.append(self.dependencies_header())
+            for dep in package['dependencies']:
+
+                if not identified_license:
+                    dep_identified_license = None
+                else:
+                    dep_identified_license = super()._get_dep_license(dep, identified_license)
+
+                version = dep.get('version', None)
+
+                output.append(self.dependency_header(dep['name']))
+                if version:
+                    output.append(f"* version: {version}")
+                output.append(f"* declared license:  {dep['license']}")
+                if dep_identified_license:
+                    output.append(f"* concluded license: {dep_identified_license}")
+                output.append("\n")
+
+            output.append(self.identified_license_header())
+            license_list = list(verification['all_licenses'])
+            license_list.sort(key=str.lower)
+            for lic in license_list:
+                output.append(f"* {lic}\n")
+
+            # Add problem discussion
+            if not dep_identified_license:
+                output.append(self.problem_header())
+                output.append("Could not conclude an outbound license for the combined work.\n")
+        return "\n".join(output)
```

Comment:
Same here. The structure of the string could be pre-defined and the actual variables should be provided by separate functions.